tests/pz_test.py

```diff
@@ -0,0 +1,16 @@
+from pettingzoo.test import api_test
+from sumo_rl import SumoEnvironmentPZ, make_env
+
+
+def test_api():
+    env = make_env(net_file='nets/4x4-Lucas/4x4.net.xml',
+                   route_file='nets/4x4-Lucas/4x4c1c2c1c2.rou.xml',
+                   out_csv_name='outputs/4x4grid/a3c',
+                   use_gui=False,
+                   num_seconds=80000)
+    api_test(env)
```

Comment:
Perhaps a render test and seed test should be added here if appropriate?

Reply:
probably, yes